src/components/Form/HistoryChainDropdown/HistoryChainDropdown.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import { BiCheckboxChecked } from 'react-icons/bi';
+import { IoIosArrowDown } from 'react-icons/io';
+
+// hooks
+import { useSelectedChains } from '../../../hooks/useSelectedChainsHistory';
+
+// utils
+import { getChainName } from '../../../utils/blockchain';
+
+const HistoryChainDropdown = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const { selectedChains, setSelectedChains } = useSelectedChains();
+
+  const allChainsOptions = [
+    {
+      chainId: 1,
+      chainName: 'Ethereum',
+    },
+    {
+      chainId: 137,
+      chainName: 'Polygon',
+    },
+    {
+      chainId: 8453,
+      chainName: 'Base',
+    },
+    {
+      chainId: 100,
+      chainName: 'Gnosis',
+    },
+  ];
```

Comment:
Does this belong in `utils/blockchain.ts`? Looks useful to be used elsewhere

Reply:
Yeah fair enough, I moved it to utils